pkg/apis/eventing/v1alpha1/register.go

```diff
@@ -45,12 +45,16 @@ var (
 // Adds the list of known types to Scheme.
 func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
```

Comment:
Prior to kubecon, we had discussed iterating on higher-level concepts in a distinct API group that we could elevate to beta, ga at its own pace. I think this is a case where that would be useful to do. How about adding this in a new API group instead of adding to `eventing`?

Reply:
With the potential of splitting eventing and messaging, this might be all that is left in eventing. Is it worthwhile making a new API group?